lib/please_cli/please_cli/base_image.py

```diff
@@ -102,6 +103,7 @@ def cmd(docker_username,
                                                  )
 
 
+    docker_repo = docker_repo or secrets['DOCKER_REPO']
```

Comment:
`secrets['DOCKER_REPO']` is never going to be used since a default value of `please_cli.config.DOCKER_REPO` is going to be used. I think at this point is best to revert change in this file, since we have to solve first where configuration is stored (#1220)